SharedCode/src/commonMain/kotlin/dataObjects/frontendDataObjects/JourneyTableDataElem.kt

```diff
@@ -0,0 +1,49 @@
+package com.jetbrains.handson.mpp.mobile.dataObjects.frontendDataObjects
+
+import com.jetbrains.handson.mpp.mobile.dataObjects.Journey
+import com.jetbrains.handson.mpp.mobile.dataObjects.Station
+import com.soywiz.klock.DateFormat
+import com.soywiz.klock.DateTimeTz
+import com.soywiz.klock.ISO8601
+import com.soywiz.klock.parse
+
+val ISODate = ISO8601.DATETIME_COMPLETE
+const val BEST_TRAIN_OPERATOR = "LNER"
+
+class JourneyTableDataElem(journey: Journey) {
+    val startStation: JourneyStation = JourneyStation(journey.originStation)
+    private val startTime: DateTimeTz =
+        ISODate.parse(journey.departureTime)
+    val endStation: JourneyStation = JourneyStation(journey.destinationStation)
+    private val endTime: DateTimeTz = ISODate.parse(journey.arrivalTime)
+    private val journeyTime: Int = journey.journeyDurationInMinutes.toInt()
+    val changes: Int = (journey.legs.size - 1)
+    private val price: Int =
+        (journey.tickets.minBy { it.priceInPennies }?.priceInPennies ?: 10000000)
```

Comment:
Why do we have 10000000? Can't we just have price be null if we don't have any prices available?